### PR TITLE
Additional query types, gene/protein summaries, reworked data fixtures

### PR DESCRIPTION
**Pull request by SFD5311**

### CODE REVIEW

File: hubmap/query_app/filters.py [hunk 1]
```diff
@@ -256,20 +260,15 @@ def get_cell_type_filter(query_params: Dict) -> Q:
...
-        cell_type_pks = cell_qs.distinct("cell_type").values_list("cell_type", flat=True)
-
-        q = Q(pk__in=cell_type_pks)
-
-        return q
-
-    if input_type == "dataset":
-
-        cell_qs = Cell.objects.filter(dataset__uuid__in=input_set)
+    entities_dict = {
```
Comment (mruffalo) [inline hubmap/query_app/filters.py:263]: I like this type of dispatch based on entity type -- much cleaner than the `if`/`elif`/... blocks.

File: hubmap/query_app/migrations/0008_delete_atacquant_delete_codexquant_and_more.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Generated by Django 4.0.6 on 2022-08-03 12:55
```
Comment (mruffalo) [inline hubmap/query_app/migrations/0008_delete_atacquant_delete_codexquant_and_more.py:1]: Did we start this app with Django 3.2? Out of curiosity, were there any backwards-incompatible changes in 4.0 that affected us?

**Review by mruffalo — APPROVED** [review]
Looks good, thanks!

File: hubmap/query_app/migrations/0008_delete_atacquant_delete_codexquant_and_more.py (hunk 2 above)
Comment (SFD5311) [reply hubmap/query_app/migrations/0008_delete_atacquant_delete_codexquant_and_more.py:1]: I think so, and none that affected us, as I recall.

**Pull request merged**